onnxruntime/core/mlas/lib/amd64/cvtfp16Avx2.asm

```diff
@@ -0,0 +1,148 @@
+;++
```

Comment:
could you please add also add a version for Linux?

Reply:
Done!